pkg/apis/build/v1alpha1/image_reconcile_build.go

```diff
@@ -58,6 +64,16 @@ func (r upToDateBuild) Apply(creator BuildCreator) (ReconciledBuild, error) {
 }
 
 func (r upToDateBuild) conditions() duckv1alpha1.Conditions {
+	if !r.builder.Ready() {
+		return duckv1alpha1.Conditions{
+			{
+				Type:   duckv1alpha1.ConditionReady,
+				Status: corev1.ConditionFalse,
+				Reason: BuilderNotReady,
```

Comment:
Super small thing here but, because this condition is largely for human consumption can we add an easy message here?